Mathlib/Data/Int/Interval.lean

```diff
@@ -129,6 +129,18 @@ theorem card_Ioc_of_le (h : a ≤ b) : (#(Ioc a b) : ℤ) = b - a := by
 theorem card_Ioo_of_lt (h : a < b) : (#(Ioo a b) : ℤ) = b - a - 1 := by
   rw [card_Ioo, sub_sub, toNat_sub_of_le h]
 
+theorem Icc_of_eq_sub_1 (h : a = b - 1) : Icc a b = {a, b} := by
```

Comment:
I would have expected `b = a + 1` as assumption. Can you please tell me why you think this form is more convenient?

Also, the name should mention `one` instead of `1`.

Reply:
Renamed it. 

@js2357, could you please address this review comment?